pyshtools/shclasses/shcoeffsgrid.py

```diff
@@ -629,6 +633,91 @@ def from_cap(self, theta, lmax, clat=None, clon=None, normalization='4pi',
                                    degrees=False)
 
         return temp
+    
+    @classmethod
+    def from_netcdf(self, filename, lmax=None, normalization='4pi', csphase=1):
+        """
+        Initialize the class with spherical harmonic coefficients from a 
+        netcdf file.
+
+        Usage
+        -----
+        x = SHCoeffs.from_netcdf(filename)
```

Comment:
add 

, [lmax, normalization, csphase])

to usage string.

Reply:
Done.